### ISSUE TITLE
Only 1 citation unit in /tlg4015/tlg001/tlg4015.tlg001.opp-grc1.xml

### ISSUE THREAD

**Issue by sonofmun** [OpenGreekAndLatin/First1KGreek#146]

**Comment by annettegessner** [comment]
commentary in Aristotle
vol. 13.1
https://archive.org/stream/commentariaina13pt12akaduoft#page/n19/mode/2up

**Comment by caitlinmiller** [comment]
@decline93 @jduff-chs @josble and I are working on this at the CHS.
The text consists of an introductory essay (p. 1-13) plus line-by-line commentary (+ app crit keyed to left-hand line numbers). There are two line numbering systems: left-hand line numbers for pages in this edition, right-hand line numbers to correspond with Ammonius' Aldine edition (1546). 
p. 14 following retains both line numbering systems, but is also divided by references to the Categories using Bekker numbers.

Our suggestion is to first divide the text between introduction and commentary, then introduce divisions by Bekker number within the commentary. The competing line number systems would be distinguished by attribute and retained for reference; but not used as organization. Working on how to implement this structure.

**Comment by josble** [comment]
For reference, there are also five text-based diagrams and charts that need to be specifically formatted in the XML. These are on pp. 28, 138, 146, 148, and 202-3.

**Comment by annettegessner** [comment]
@PonteIneptique, @sonofmun : Have we ever dealt with things like this? 
Links:
diagram on [page 28](https://archive.org/stream/commentariaina13pt12akaduoft#page/n49/mode/2up), 
diagram on [page 138](https://archive.org/stream/commentariaina13pt12akaduoft#page/n159/mode/2up), 
list on [page 146](https://archive.org/stream/commentariaina13pt12akaduoft#page/n167/mode/2up), 
diagram on [page 148](https://archive.org/stream/commentariaina13pt12akaduoft#page/n169/mode/2up) and
diagram on [page 202](https://archive.org/stream/commentariaina13pt12akaduoft#page/n223/mode/2up).

**Comment by annettegessner** [comment]
At some point it would be best to insert an image taken from the scan whenever we encounter diagrams etc. like this. So right now, we need to mark up instances like this, so that we can find them later. 
Matt and I would suggest to insert a link to the exact page in the scan to the XML. What I found at the TEI guidelines about graphics is the following:
```xml
<figure>
 <graphic url="fig1.png"/>
 <head>Figure One: The View from the Bridge</head>
 <figDesc>A Whistleresque view showing four or five sailing boats in the foreground, and a
   series of buoys strung out between them.</figDesc>
</figure>
```
We'd keep head and figDesc empty for now, but the link to the exact page will help extracting the figure at a later point.
So I would take the links I already copied in my last comment and put them in the XML, e.g.:
```xml
<figure>
 <graphic url="fhttps://archive.org/stream/commentariaina13pt12akaduoft#page/n49/mode/2up"/>
 <head></head>
 <figDesc></figDesc>
</figure>
```
Are you ok with this, @PonteIneptique ?

**Comment by PonteIneptique** [comment]
I think it should be converted to SVG on long term (but it requires some work) , you could keep the text in figDesc if necessary

```xml
<figure><svg xmlns="http://www.w3.org/2000/svg" width="6cm" height="5cm" viewBox="6 3 6 5">
<ellipse style="fill: #ffffff"
cx="9.75" cy="6.35" rx="2.75" ry="2.35"/>
</svg>
</figure>
```

> http://tei.it.ox.ac.uk/Talks/2007-10-31-MMWorkshop/P4toP5-1.xhtml

**Comment by sonofmun** [comment]
Right now, we only want to point to the page where the figure is located on the archive.org site. Does all the information about the SVG figure make sense at this early stage?

**Comment by PonteIneptique** [comment]
As I said, on long term. Through, because this is really specific to what seems to me like math/geometry/geographic, it might be interesting to ask some student to make it

**Comment by sonofmun** [comment]
So, at this point, simply adding the XML that Annette suggested above should be sufficient. Then, when we have an SVG, then we can add the more complex markup that you suggest. Correct?

**Comment by PonteIneptique** [comment]
Yes but still transcribe the text of the fig as the content of the XML annette suggested

**Comment by PonteIneptique** [comment]
The list can be transcribed in TEI ( http://www.tei-c.org/release/doc/tei-p5-doc/fr/html/ref-table.html ). I cannot find it in epidoc. But there should be way to express the table.

**Comment by PonteIneptique** [comment]
Probably with layouts.

**Comment by annettegessner** [comment]
Ok: So what can be done right now is to mark those text passages and enable later further work: 
I would ask the person in charge of this file to mark the passage as figure and include the text, which is inside the figure into figDesc as well as a link to the correct PDF-page. 

But sadly this text will often be incorrect, incomplete and/or in no logical order, so I guess, it has to be transcribed correctly and the layout should probably be preserved too.

Lets take diagram on [page 28](https://archive.org/stream/commentariaina13pt12akaduoft#page/n49/mode/2up) as an example. It would look like this: 
```xml 
<figure>
 <graphic url="https://archive.org/stream/commentariaina13pt12akaduoft#page/n49/mode/2up"/>
 <figDesc>οὐσία
συμβεβηκός
οὐκ ἐν
δποκειμένω
καὶ
καθ’ δπο
κειμενου
κα
θόλου
ἀσύστατον
μερ
ἐν ὕπο.
κειμένῳ
καὶ οὐ
καθ’ δπο-
κειμενου
ἷκόν</figDesc>
</figure>
```
This uncorrected text taken directly from the XML has several problems: 
- OCR-errors: For instance the word δποκειμένω is spelled incorrectly for it should be ὑποκειμένω. Normally OCR errors like this could be taken care of at a later point for all the texts, but maybe it's better to take extra care of these texts right now.
- Layout: Another problem is the word order is all messed up and does not represent, what the reader should read i.e. what words belong together. Should we try to fix this now or leave it to the person doing the SVG later? If yes: How should we mark the text in this case? Set p-tags around every single textpart?

Would it be alright to ask the CHS students to try to find a solution for these texts inside the figures and ask Thibault, if they have questions?

**Comment by PonteIneptique** [comment]
This is right. Except for list which are not technically figure :)

**Comment by annettegessner** [comment]
I tried summarizing this issue in #251, where we can add or correct ad libitum.

**Comment by josble** [comment]
If we're interested in preserving layout in the short-term, I think it makes sense to format at least some of the diagrams as tables rather than as figures. Here's an example table for the diagram on p. 148:
<img width="637" alt="screen shot 2016-06-14 at 3 46 39 pm" src="https://cloud.githubusercontent.com/assets/19731628/16057551/a02ce14e-3248-11e6-9f6f-4127795bcaac.png">

One issue with that approach: despite correctly displaying texts formatted using the <table> tag (e.g. the screenshot above), oXygen won't allow the element "table" anywhere in the file. Couldn't get <layout> to work either. Haven't been able to find a solution online; any suggestions for retaining this formatting?

**Comment by annettegessner** [comment]
Strange! Could there be a problem with Epidoc, @PonteIneptique ?

**Comment by PonteIneptique** [comment]
I think when we can we should always have both (figure and layout) so that we can have the text in the TEI and have the figure at the same time, probably using `@sameAs` attribute on the `layout` tag and `xml:id` on the svg one like : 

```xml
<div>
<svg xml:id="">
<rect x="1">Text 1</rect>
<rect x="1">Text 2</rect>
</svg>
<layout>
</layout>
</div>
```

I think you should contact Simona or someone from epidoc for the table best practice.

**Comment by annettegessner** [comment]
Hello @simonastoyanova ,
Can you help some former colleages out with your Epidoc magic? ;-) We are thinking about the best practive for tables and/or figures. You should find all information necessary in this issue and a summary in #251. Thanks!

**Comment by annettegessner** [comment]
Simona just wrote me an email:

"What we've done for tables, diagrams and images has been to link to the image provided during the OCR process, as I think you suggest in github. We can't adequately mark them up in xml, and so we just break the flowing text and put a link to the image with the table/diagram. That's a short-term thing, though. Depending on how deep you want your encoding and what functionality you want to provide around these diagrams, it might be a good idea to do something like what Thibault suggests, but that would require a lot of work (and time, at first). That's what I found in the TEI guidelines: http://www.tei-c.org/release/doc/tei-p5-doc/en/html/FT.html 
and http://www.tei-c.org/release/doc/tei-p5-doc/en/html/GD.html.

However, this is not allowed in EpiDoc, so you will have to come up with a Perseus-specific schema that is based on the EpiDoc one but also includes some TEI elements etc. which are not part of EpiDoc. This has been an ongoing discussion for a long time, though I'm not sure what's the progress there, and it might be best to ask Bridget, see if she remembers.

Sorry I can't be more helpful, but these are things that don't appear in epigraphy, so they're not provided for in the EpiDoc schema. I guess, first decide what exactly you want to use them for, and then figure out how best to achieve that. I'm thinking the Perseus-specific schema would be the way to go (to also take care of other structural elements that appear in paper editions but not in epigraphy)."
